packages/factory/package.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "@dd/factory",
```

Comment:
Do we have the rights on `@dd`?

Reply:
This is not published, only internal dependencies.